generate_datafiles/compound_disease_datafiles.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
```

Comment:
Just to clarify, so you'll have one file like this per edge type?

Reply:
Sort of. It is more like one file per node group i.e. (Compound Disease nodes and all the edges between them or Disease Gene nodes and all the edges between them. [Full picture here](https://raw.githubusercontent.com/dhimmel/rephetio/5f834b14b94b9b9d2082c5ae0303b57d634c3a40/figure/metagraph-cypher.png) to give you a better idea)